### ISSUE TITLE
Add term hemimelia of forearm

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10493]
**Preferred term label:**
Hemimelia of forearm

**Synonyms**

**Definition (free text, please give PubMed ID)**
A developmental defect leading to a transverse deficiency with absence of the forearm (eg ulna and radius), wrist and hand.
PMID: [32954639](https://pubmed.ncbi.nlm.nih.gov/32954639), ORPHA:294979

**Parent term (use hpo.jax.org/app)**
Transverse terminal upper limb defect (#10328) 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Adams-Oliver syndrome (OMIM100300)

**Your nano-attribution (ORCID)**
A. Zankl 0000-0001-8612-1062, @azankl
L. Wit 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@azankl
Adding new term: 
- ID: HP:6000948
- label: Hemimelia of forearm
- parents: [Transverse terminal limb defect (HP:6000818)](https://hpo.jax.org/app/browse/term/HP:6000818)
- pmid: PMID:32954639
- def: A developmental defect leading to a transverse deficiency with absence of the forearm (eg ulna and radius), wrist and hand.
- comment: 
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10493
- synonyms:
